lib/rspec/core/lookup_set.rb

```diff
@@ -0,0 +1,32 @@
+module RSpec
+  module Core
+    # @private
+    class LookupSet
```

Comment:
I guess I'm just overly cautious given my knowledge of Rails dubious autoloading lol :)